domain/src/main/java/org/sagebionetworks/research/domain/repository/TaskRepository.java

```diff
@@ -73,4 +75,13 @@
     @NonNull
     @CheckResult
     Completable setTaskResult(TaskResult taskResult);
+
+    /**
+     * Finds and returns the drawable resource with the given identifier, or throws a NotFoundException()
+     * @param name - the identifier of the drawable to find.
+     * @return the drawable resource with the given identifier.
+     * @throws NotFoundException if the given resource identifier cannot be resolved as a drawable.
+     */
+    @DrawableRes
+    int resolveDrawableFromString(@NonNull String name) throws NotFoundException;
```

Comment:
Does this need to take a type? IIRC we talked about drawable types like remote vs resource